ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/project/node/remove/DeleteNodeHandler.java

```diff
@@ -0,0 +1,408 @@
+/*******************************************************************************
```

Comment:
At this moment I blocked with creating test for this class because methods bellow are use static methods for Promises. And mockito testing library doesn't support mock static methods (http://stackoverflow.com/a/4482364). We may use PowerMock for testing or refactor Promises class to extract interface from it and make it injectable to allow writing tests for other stuff.